src/action.ts

```diff
@@ -62,13 +63,11 @@ async function fetchOrLocateDefectDojoResultsFile() {
   return fetchOrLocateResultsFile("defectdojo", results, fileName);
 }
 
-async function fetchOrLocateContrastResultsFile() {
-  let file = core.getInput("file");
-  if (file !== "") {
-    return file;
-  }
+async function fetchOrLocateContrastResultsFile(tool: Tool) {
```

Comment:
I think you no longer need to pass the Tool, you can pass an String to the fetchOrLocateResultsFile